pywbemtools/pywbemcli/_common.py

```diff
@@ -456,6 +456,51 @@ def create_cimproperty(cim_type, is_array, name, value_str):
     return CIMProperty(name, cim_value, cim_type)
 
 
+def create_ciminstancename(cim_class, kv_keys):
+    """
+    Create a cim instance from the input parameters.
+
+      Parameters:
+
+        cim_class_: (:Class:`pywbem.CIMClass`)
+            The class from which the CIMInstance is to be created
+
+        properties ():
+            A tuple of name/value pairs representing the properties and their
+            values that are to be constructed for the instance. Required
```

Comment:
Adjust the parameter description to the actual parameter.

Reply:
DONE